cmd/world/cardinal/dev.go

```diff
@@ -83,6 +83,9 @@ var devCmd = &cobra.Command{
 			cmdErr <- err
 		}()
 
+		fmt.Println("Cardinal Started..")
+		fmt.Println("Log is suppressed, use --debug to see the logs")
```

Comment:
This is only for the World CLI logs right? I don't think we need this message, it just confuses user because it makes them think the cardinal logs are suppressed

Reply:
not only the CLI logs, but it will suppress stderr from docker command also

Reply:
Don't we want those logs?

Reply:
At the first, I thought the World-CLI user would be an everyday person rather than someone with a developer background. However, upon reconsideration, it seems more fitting for the user to be a game developer. 
Yes, if the user has a developer background, they definitely need access to the logs. 
I believe I overlooked this aspect.

i will review the logs that should be released and those that should not be released in normal mode.

Reply:
removed this println, and the adjustment of the log will coming into the next PR